dvc/stage.py

```diff
@@ -326,6 +331,17 @@ def reproduce(
 
         return self
 
+    def update(self):
+        for dep in self.deps:
+            dep.update()
+
+        locked = self.locked
+        self.locked = False
+        try:
+            self.reproduce()
```

Comment:
can we reuse `force` or something, to avoid this locked flag manipulation ... does not look really nice

Reply:
`force` doesn't affect `locked` stages. But I'll take a look.